examples/adjoints/adjoints_ex7/femparameters.C

```diff
@@ -0,0 +1,753 @@
+
+#include "femparameters.h"
```

Comment:
Does this differ from the adjoints_ex1 femparameters.C?  If not, could we just use the same symlink trick that we use in the other adjoints examples?

Reply:
Done.